src/chapters/tableau-part4/studio.rst

```diff
@@ -1,2 +1,110 @@
 Studio: Tableau Part 4 - Stories and Dates
-==========================================
\ No newline at end of file
+==========================================
+
+Getting Started
+---------------
+
+| This is a group studio.  Your group has been asked to create a visual for the International Fencing Federation (FIE).   Work through the data analysis steps that you have been learning in this program to create your final viz. 
+
+| As a group, explore your data and decide upon: 
+
+1. A business issue or a question you want to explore further.  
+#. Next clean, and manipulate the data to create vizzes that provide insight to your business issue or question.  
+#. Your final output should be a dashboard that best provides insight on your discoveries.  
+#. You will present your dashboard at the end of class.
+#. Suggested Topics to Explore:
+
+   A. Weapons
+   #. Handedness
+   #. Category
+   #. Ages
+   #. Location and/or Time Zone
+   #. What factors predict wins? 
+   
+| You are not limited to these topics.  If you discover something more interesting, explore away!  
+
+Setting Up the Studio:
+^^^^^^^^^^^^^^^^^^^^^^
+
+Download `FIE Fencing Womens Foil data set <https://www.kaggle.com/datasets/amichaelsen/fie-fencing-womens-foil-data>`__. There are 4 csv files. To download all csv files at once, click on the "Download" bubble next to the "New Notebook" bubble.  An orange arrow is pointing to the "Download" bubble in the image below.
+
+.. figure:: figures/download-multi-files.png
+   :alt: An orange arrow points to the "Download" bubble, highlighting the Download option of kaggle.
```

Comment:
We have gone over this before with students so why repeat it?

Reply:
We have repeated it in other chapters.  Originally done in canvas to help students.